src/django/pfb_analysis/models.py

```diff
@@ -43,8 +43,11 @@ def __repr__(self):
 
     def save(self, *args, **kwargs):
         """ Override to do validation checks before saving, which disallows blank state_abbrev """
+        if not self.label:
+            self.label = slugify(self.name)
         self.full_clean()
-        super(Neighborhood, self).save(*args, **kwargs)
+        neighborhood = super(Neighborhood, self).save(*args, **kwargs)
+        return neighborhood
```

Comment:
What's the reason for this. The Django model API spec indicates that `save()` shouldn't return anything.